src/it/java/com/google/checkstyle/test/chapter5naming/rule523methodnames/MethodNameTest.java

```diff
@@ -37,7 +37,7 @@ protected String getPackageLocation() {
     public void testMethodName() throws Exception {
         final Configuration checkConfig = getModuleConfig("MethodName");
         final String msgKey = "name.invalidPattern";
-        final String format = "^[a-z][a-z0-9][a-zA-Z0-9_]*$";
+        final String format = "^[a-z][a-z0-9]\\w*$";
```

Comment:
Is `_` is a `\w` ?

Reply:
Yes. https://www3.ntu.edu.sg/home/ehchua/programming/howto/Regexe.html
> \w, \W: ANY ONE word/non-word character. For ASCII, word characters are [a-zA-Z0-9_]

Reply:
Surprised, but good to know